docs/client/install-kn.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Installing the kn CLI"
+weight: 10
+type: "docs"
+aliases:
+  - /docs/install/install-kn
+  - /docs/client/connecting-kn-to-your-cluster
+---
+
+This guide provides details about how you can install the Knative `kn` client (CLI) using various methods.
+
+## Install kn using an executable binary
+
+You can install `kn` by downloading the executable binary for your system and placing it in the system path.
+
+Links to the latest stable executable binary releases are available on the <a href="https://github.com/knative/client/releases" target="_blank">`kn` release page</a>.
+
+
+### Install kn using nightly executable binary
+
+Nightly executable binaries are available for users who want to install the latest pre-release build of `kn`.
+
+**WARNING:** Nightly executable binaries include features which may not be included in the latest Knative release and are not considered to be stable.
+
+Links to the latest nightly executable binaries are available here:
+
+- <a href="https://storage.googleapis.com/knative-nightly/client/latest/kn-darwin-amd64" target="_blank">macOS</a>
+- <a href="https://storage.googleapis.com/knative-nightly/client/latest/kn-linux-amd64" target="_blank">Linux</a>
+- <a href="https://storage.googleapis.com/knative-nightly/client/latest/kn-windows-amd64.exe" target="_blank">Windows</a>
+
+## Install kn using Go
+
+1. Check out the `kn` client repository:
+
+      ```
+      git clone https://github.com/knative/client.git
+      cd client/
+      ```
+
+1. Build an executable binary:
+
+      ```
+      hack/build.sh -f
+      ```
+
+1. Move `kn` into your system path, and verify that `kn` commands are working properly. For example:
+
+      ```
+      kn version
+      ```
+
+## Install kn using brew
+
+For macOs, you can install `kn` by using <a href="https://github.com/knative/homebrew-client" target="_blank">brew</a>.
+
+## Installing kn using container images
```

Comment:
```suggestion
## Run kn using container images
```

Reply:
How is "running" different to installing it? If there's a difference here we should explain it, otherwise it seems odd to have commands to "run" something in the install section

Reply:
"Installing" means putting the binary into a permanent place on your system. "Running" it doesn't do it but just executes it. Next time to run it this way you again have potentially download the image again.

The general concept of containers is that you can run code without installing it locally. Therefore you don't use "installing a container image" but "running a container" when describing this action.

Maybe just switch the section title from "installing" to "using" or "starting" ? I think the user is mostly concerned about how she can *run* `kn`, regardless of whether it is installed locally or started from within a container image (or maybe even run with `go run` which would be even another option).

Reply:
That was kind of what I thought but thank you for clarifying it :) Yeah I think it makes sense maybe to change it to "using kn" rather than installing in this context, will give this an update.

Reply:
Maybe actually something like "setting up" or "getting started with kn"

Reply:
> Next time to run it this way you again have potentially download the image again.

If we call this out I think it makes the difference clearer too